lib/challenge.rb

```diff
@@ -1,7 +1,36 @@
+# @Author: xiaojiezhang
+# @Date:   2019-01-24T09:00:48-05:00
+# @Last modified by:   xiaojiezhang
+# @Last modified time: 2019-01-24T09:30:48-05:00
+
+
+require 'pry'
 # frozen_string_literal: true
 
 class FibonacciRange
+  include Enumerable
+
+  def each
+  current_fib =0
+  next_fib=1
+    while current_fib < @stop
+        yield current_fib
+        previous = current_fib
+        current_fib = next_fib
+        next_fib= current_fib+previous
+    end
+  end
```

Comment:
🎉 Very clean variable names!

🌱 Style: Indentation is inconsistent. The linter can give you hints at how to improve your indentation. You could also run `Packages > Atom Beautify > Beautify`, to clean up your indentation. 🙂